brave/brave-core
# Remove third_party-crashpad-crashpad-client-settings.cc.patch

This patch could be removed in favour of a chromium_src override
redefining the Settings::GetClientID() method into something that
uses the original implementation from upstream and simply resets
the memory of client_id before returning when needed.

Resolves https://github.com/brave/brave-browser/issues/14789

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

Based on description of https://github.com/brave/brave-browser/issues/8865:

1. go to brave://crash
2. go to brave://crashes and ensure the crash has been sent
3. lookup the crash id on backtrace
4. guid should be 0 or not present
